bndtools.core/src/bndtools/launch/BndDependencySourceContainer.java

```diff
@@ -98,11 +100,22 @@ public ISourceContainerType getType() {
                         }
                     }
                 }
+
+                lastRun = run;
```

Comment:
Looking at the type hierarchy of eclipse source containers I don't believe this will ever get called a 2nd time where lastRun could be null, but to make it more resilient to behavior changes in Ecilpse, we dispose of previous run if someone calls "createSourceContainers" again.